md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessDependencyAPI.java

```diff
@@ -46,14 +47,13 @@
      * @return restWrapper It contains an instance of LineageInfo.
      */
     @RequestMapping(value = {"", "/"}, method = RequestMethod.GET)
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper get(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "get" is 14 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)